core/src/test/scala/unit/kafka/utils/CommandLineUtilsTest.scala

```diff
@@ -52,4 +52,13 @@ class CommandLineUtilsTest {
     assertEquals("Value of second property should be 'second'",props.getProperty("second.property"),"second")
   }
 
+  @Test
+  def testParseArgsWithMultipleDelimiters() {
+    val argArray = Array("first.property==first","second.property=second=", "third.property=thi=rd")
+    val props = CommandLineUtils.parseKeyValueArgs(argArray, false)
+    assertEquals("Value of first property should be '=first'",props.getProperty("first.property"),"=first")
```

Comment:
nit: add a space between method arguments here and other places

Reply:
added 👍